subcommand/acl-controller/command_test.go

```diff
@@ -502,18 +511,12 @@ func TestUpsertAnonymousTokenPolicy(t *testing.T) {
 			},
 			expPolicy: expOSSAnonTokenPolicy,
 		},
-		"mgw WAN fed enabled, primary DC, create policy ENT": {
-			agentConfig: AgentConfig{
-				Config: Config{Datacenter: "dc1"},
-				DebugConfig: Config{
-					PrimaryDatacenter:               "dc1",
-					MeshGatewayWANFederationEnabled: true,
-				},
-			},
-			partitionsEnabled: true,
-			expPolicy:         expEntAnonTokenPolicy,
-		},
-	}
```

Comment:
This I moved to the `command_ent_test.go` so that in doesn't run for the OSS binary, since the controller now passes in an explicit `partition=default` query options.